src/About.jsx

```diff
@@ -1,15 +1,34 @@
-import React from 'react';
-import {Container, Row, Col} from 'react-bootstrap';
+import React from "react";
+import { Container, Row, Col } from "react-bootstrap";
 
-export const About = ()=> {
-    return (
-        <Container><Row><Col>
-            <h2>About Noodle</h2>
-            
-            <p>Scheduling events <em>used</em> to be easy on the other sites. Then they started to suck. Intrusive ads, pop-ups, forced logins, slow-loading pages, all that jazz.</p>
+export const About = () => {
+  return (
+    <Container>
+      <Row>
+        <Col>
```

Comment:
I believe the `<Container>`, `<Row>`, and `<Col>` that this is inside lives in the App.js file, so you shouldn't need these tags.